### ISSUE TITLE
[Bug]: enabling "delete_without_reviewing" causes error X-2

### ISSUE THREAD

**Issue by McNuggetSmuggler** [ThioJoe/YT-Spammer-Purge#448]
### Duplicate Issues

- [X] There are **no existing posts relating to my problem**
- [ ] There are **existing posts relating to my problem, but the solution given, doesn't work for me.**

### What happened?

I enabled "delete without reviewing" in the config file and scanned a video. Everything worked fine until the scan finished by which point the error showed its ugly face. Disabling "delete without reviewing" gets rid of the issue. 

### Release version

2.14.0

### Steps to reproduce

1. Enabled "delete_without_reviewing" in the config file
2. scanned a video from someone else's channel in Auto-Smart Mode

### What platform are you seeing this problem on?

Windows (.exe file)

### Relevant log output

```shell
Traceback (most recent call last):
  File "D:\Users\Joe\Documents\GitHub ThioJoe\YouTube-Spammer-Purge\YTSpammerPurge.py", line 1478, in <module>
  File "D:\Users\Joe\Documents\GitHub ThioJoe\YouTube-Spammer-Purge\YTSpammerPurge.py", line 1455, in main
  File "D:\Users\Joe\Documents\GitHub ThioJoe\YouTube-Spammer-Purge\YTSpammerPurge.py", line 1363, in primaryInstance
UnboundLocalError: local variable 'exclude' referenced before assignment
------------------------------------------------
Error Message: local variable 'exclude' referenced before assignment

Error - Code: X-2 occurred. This is almost definitely my fault and requires patching (big bruh moment)
```

### Screenshots

![bug report snap](https://user-images.githubusercontent.com/98851699/152055985-dcc203c9-bbef-482c-8f93-be7ed856e2fb.PNG)

**Comment by TechStudent11** [comment]
> ```shell
> Traceback (most recent call last):
>   File "D:\Users\Joe\Documents\GitHub ThioJoe\YouTube-Spammer-Purge\YTSpammerPurge.py", line 1478, in <module>
>   File "D:\Users\Joe\Documents\GitHub ThioJoe\YouTube-Spammer-Purge\YTSpammerPurge.py", line 1455, in main
>   File "D:\Users\Joe\Documents\GitHub ThioJoe\YouTube-Spammer-Purge\YTSpammerPurge.py", line 1363, in primaryInstance
> UnboundLocalError: local variable 'exclude' referenced before assignment
> ------------------------------------------------
> Error Message: local variable 'exclude' referenced before assignment
> 
> Error - Code: X-2 occurred. This is almost definitely my fault and requires patching (big bruh moment)
> ```

I can see the issue.

I think I can fix it.

**Comment by ThioJoe** [comment]
Ah ok yea these errors are relatively simple to solve, it means I forgot to account for a certain situation. I'm working on a patch for a few other bugs so I'll do this one too

**Comment by TechStudent11** [comment]
> Ah ok yea these errors are relatively simple to solve, it means I forgot to account for a certain situation. I'm working on a patch for a few other bugs so I'll do this one too

ok. i was gonna create a pr but it looks like you have it covered

**Comment by ThioJoe** [comment]
Fixed, it will be in the next release 14.1 in a bit

**Comment by ThioJoe** [comment]
Ok it's fixed in the latest 2.14.2 release
